docs/userguides/httpbin.md

```diff
@@ -0,0 +1,107 @@
+<!--
```

Comment:
We can modify this file name to `apisix_route.md`, from the article, it mainly introduces the CRD ApisixRoute.

We can add  apisix_service.md and Apisix_upstream.md in future.

Reply:
I don't think we should name a document by the stuff we use, on the contrary, we should name it by what it does.

Reply:
The `httpbin` is a upstream demo, but the `userguides` path need to give user some guides about how to use ingress-controller.
With `ApisixRoute` in this file we define simple route. But the guide of ingress-controller is not only `ApisixRoute`, and we can all use `httpbin` for demo.

Reply:
Maybe we should use `example` as the dir.